RubeesEat/Pages/BillDetails.cshtml

```diff
@@ -15,19 +15,19 @@
     {
         var item = billRepository.GetById(billId);
         <div class="bill-summary">
-            <div class="description">@item.Description</div>
-            <div class="personWhoPayed">Paid by: @(item.EntryLines[0].Person.FirstName)
+            <div class="description" t-id="description">@item.Description</div>
+            <div class="personWhoPaid" t-id="personWhoPaid">Paid by: @(item.EntryLines[0].Person.FirstName)
                 @(item.EntryLines[0].Person.LastName) </div>
-            <div class="date">@item.Date.ToString("D")</div>
-            <div class="amount">Total amount: @item.EntryLines[0].Amount€</div>
+            <div class="date" t-id="date">@item.Date.ToString("D")</div>
+            <div class="amount" t-id="amount">Total amount: @item.EntryLines[0].Amount€</div>
```

Comment:
Ok before we forget removing the class names later on: Please remove all the other classes here and remove the CSS rule. We only set the margin so it might look bad for now but we will fix this with bootstrap later. This way, we won't forget removing these rules later on.